From: Carlos Maddela <e7appew@gmail.com>
Date: Mon, 19 Dec 2016 00:58:24 +1100
Subject: Fix format security warnings.

Description: Fix format security warnings.
 They are treated as errors in Standards Version 3.9.8.
Author: Carlos Maddela <e7appew@gmail.com>
Origin: vendor
Forwarded: no
Last-Update: 2016-12-19
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
---
 tools/memory.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/memory.c b/tools/memory.c
index f2c0c48..edf32ec 100644
--- a/tools/memory.c
+++ b/tools/memory.c
@@ -165,7 +165,7 @@ static void print_ptr(memchunk *mc, int size)
 static void print_ptrs(char *msg)
 {  int bucket,i,n=0;
 
-   fprintf(stdout, msg);
+   fprintf(stdout, "%s", msg);
 
    for(bucket=0; bucket<64; bucket++)
       for(i=0; i<phCnt[bucket]; i++)
